packages/client/lib/sync/fullsync.ts

```diff
@@ -208,12 +208,32 @@ export class FullSynchronizer extends Synchronizer {
     const baseFeeAdd = this.config.chainCommon.gteHardfork(Hardfork.London)
       ? `baseFee=${blocks[0].header.baseFeePerGas} `
       : ''
+
+    let attentionHF: string | null = null
+    const nextHFBlockNum = this.config.chainCommon.nextHardforkBlockBN()
+    if (nextHFBlockNum !== null) {
+      const remaining = nextHFBlockNum.sub(last)
+      if (remaining.lten(10000)) {
+        const nextHF = this.config.chainCommon.getHardforkByBlockNumber(nextHFBlockNum)
+        attentionHF = `${nextHF} HF in ${remaining} blocks`
+      }
+    } else {
+      if (this.config.chainCommon.hardfork() === Hardfork.PreMerge) {
+        const mergeTD = this.config.chainCommon.hardforkTD(Hardfork.Merge)
+        const td = this.chain.blocks.td
+        const remaining = mergeTD!.sub(td)
+        if (remaining.lten(50000000000)) {
+          attentionHF = `Merge HF in ${remaining} TD (diff)`
```

Comment:
If I read this code correctly, it looks like we should see this attention banner within 5MM total difficulty of TTD and I'm not seeing it yet and we're within that 5MM range (kiln having a TTD of 20,000,000)

![image](https://user-images.githubusercontent.com/17355484/159026082-b3e7873a-adb2-4f82-aad4-3295193f4ad1.png)

Reply:
wow that's weird, I tried this as a test:

```typescript
console.log(remaining.lten(20000000000000), remaining.lte(new BN(20000000000000)))
```

output: `false true`

how could that be?

Reply:
<img width="1714" alt="Screen Shot 2022-03-18 at 2 32 46 PM" src="https://user-images.githubusercontent.com/22116/159085709-ed9cf7bb-36e1-4f06-ac89-6706127f995c.png">

ok cool it works now, but i think we should set it back quite a bit more

currently it is `50000000000`, ttd is `20000000000000`. should we say when it's 10% away? so once td reaches `18000000000000`. in `kiln` that would be block number `53623` (merge reached at `55127`)